web/gds-admin-ui/src/constants/dashboard.js

```diff
@@ -3,4 +3,16 @@ export const Status = {
     PENDING_REVIEW: "PENDING REVIEW",
     VERIFIED: "VERIFIED",
     SUBMITTED: "SUBMITTED"
+}
+
+export const STACKED_BAR_LABEL = {
+    SUBMITTED: "SUBMITTED",
+    EMAIL_VERIFIED: "EMAIL_VERIFIED",
+    PENDING_REVIEW: "PENDING_REVIEW",
+    REVIEWED: "REVIEWED",
+    ISSUING_CERTIFICATE: "ISSUING_CERTIFICATE",
+    VERIFIED: "VERIFIED",
+    REJECTED: "REJECTED",
+    APPEALED: "APPEALED",
+    ERRORED: "ERRORED"
```

Comment:
I haven't gotten too far yet in the PR; but note that these should be identical to the `Status` constant above. With that in mind; is it possible to use the `Status` constant for data parsing and to transform the labels into something human readable? E.g. `Issuing Certificate` instead of `ISSUING_CERTIFICATE` (title case, with spaces instead of underscore).

Reply:
@elysee15 so it looks like this hasn't happened yet? My recommendation is as follows:

```javascript
export const Status = {
    SUBMITTED: "SUBMITTED",
    EMAIL_VERIFIED: "EMAIL_VERIFIED",
    PENDING_REVIEW: "PENDING_REVIEW",
    REVIEWED: "REVIEWED",
    ISSUING_CERTIFICATE: "ISSUING_CERTIFICATE",
    VERIFIED: "VERIFIED",
    REJECTED: "REJECTED",
    APPEALED: "APPEALED",
    ERRORED: "ERRORED"
}

export const StatusLabel = {
    SUBMITTED: "Submitted",
    EMAIL_VERIFIED: "Email Verified",
    PENDING_REVIEW: "Pending Review",
    REVIEWED: "Reviewed",
    ISSUING_CERTIFICATE: "Issuing Certificate",
    VERIFIED: "Verified",
    REJECTED: "Rejected",
    APPEALED: "Appealed",
    ERRORED: "Errored"

export const STACKED_BAR_LABEL = StatusLabel
```

You can then string manipulate StatusLabel however you'd like, e.g. toupper().